Sources/CoreAPI/WordPressOrgXMLRPCApiError.swift

```diff
@@ -0,0 +1,40 @@
+import Foundation
+
+public struct WordPressOrgXMLRPCApiError: Error {
+
+    /// Error constants for the WordPress XML-RPC API
+    @objc public enum Code: Int, CaseIterable {
+        /// An error HTTP status code was returned.
+        case httpErrorStatusCode
+        /// The serialization of the request failed.
+        case requestSerializationFailed
+        /// The serialization of the response failed.
+        case responseSerializationFailed
+        /// An unknown error occurred.
+        case unknown
+    }
+
+    let code: Code
+}
```

Comment:
@crazytonyli I ended up having to wrap the error code enum into an `Error` type like you did for the REST API error.

Otherwise, between the `@objc` visibility (needed because WordPress-Authenticator uses it 🙄 ) and the `Error` conformance, I couldn't find a way to stop the compiler from automatically generating `WordPressOrgXMLRPCApiErrorDomain` for `@objc public enum WordPressOrgXMLRPCApiError: Int, Error`.

I think this is an okay solution. Apart for the fact that's the only one I could get to work, it's, as far as I can see, the same setup as the other error, which makes the implementation consistent.